projects/plugins/jetpack/modules/google-fonts/current/load-google-fonts.php

```diff
@@ -94,8 +101,12 @@ function jetpack_get_theme_fonts_map() {
 
 	$theme_fonts_map = array();
 	foreach ( $raw_data['settings']['typography']['fontFamilies'] as $font_family ) {
-		if ( isset( $font_family['name'] ) ) {
-			$theme_fonts_map[ $font_family['name'] ] = true;
+		$font_family_name = isset( $font_family['name'] )
+			? $font_family['name']
+			: Jetpack_Google_Font_Face::get_font_family_name( $font_family );
```

Comment:
The good news is, since Jetpack now requires PHP 7+, we can use the null coalescing operator :)

```suggestion
		$font_family_name = $font_family['name'] ?? Jetpack_Google_Font_Face::get_font_family_name( $font_family );
```

Reply:
Oh, nice! I saw Jetpack still supported PHP 5.6+ last time